gradle/libs.versions.toml

```diff
@@ -170,12 +173,11 @@ hamcrest-junit = { module = "org.hamcrest:hamcrest-junit", version.ref = "hamcre
 
 icu4j = { module = "com.ibm.icu:icu4j", version.ref = "icu4j" }
 
-jacoco-agent = { module = "org.jacoco:org.jacoco.agent", version.ref = "jacoco" }
 junit4 = { module = "junit:junit", version.ref = "junit4" }
 
 javax-annotation-api = { module = "javax.annotation:javax.annotation-api", version.ref = "javax-annotation-api" }
 javax-annotation-jsr250-api = { module = "javax.annotation:jsr250-api", version.ref = "javax-annotation-jsr250-api" }
-javax-inject = { module = "javax.inject:javax.inject", version.ref = "javax.inject" }
+javax-inject = { module = "javax.inject:javax.inject", version.ref = "javax-inject" }
```

Comment:
The `javax-inject` version was already defined in this file:
https://github.com/robolectric/robolectric/blob/3da6da2dc9a19c667e80a240f67585b8502e9e8c/gradle/libs.versions.toml#L72

But until this PR, the version was referenced as `javax.inject` (using a `.`), instead of `javax-inject` (using a `-`). So a warning was reported:
<img width="373" alt="Screenshot 2024-02-07 at 14 44 00" src="https://github.com/robolectric/robolectric/assets/1009664/34c53409-eab7-4fb3-a0ee-e3f86ae378ec">